From: wuyifeng <wuyifeng@nd.com.cn>
Date: Thu, 19 Jan 2023 09:43:54 +0800
Subject: Revert Enable-SPPI-for-devices-with-enough-memory.patch

Fix version: 1.4.0
The 'Enable-SPPI-for-devices-with-enough-memory.patch' will cause the issue that Drag
and Drop function doesn't work, so revert it.
Relate to https://prometheanworld.atlassian.net/browse/AP9-11548.
Relate to https://prometheanworld.atlassian.net/wiki/spaces/AP9/pages/14429487423/SPIKE+-+Drag+and+Drop+function+in+Chromium+Browser+does+not+work.
Relate to Revert-Enable-StrictOriginIsolation-and-SitePerProcess.patch, Disable-transferring-link-and-image-of-Drag-Drop-function.patch.

---
 chrome/common/chrome_features.cc      | 8 ++++++--
 components/site_isolation/features.cc | 4 ++--
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/chrome/common/chrome_features.cc b/chrome/common/chrome_features.cc
--- a/chrome/common/chrome_features.cc
+++ b/chrome/common/chrome_features.cc
@@ -1037,8 +1037,12 @@ BASE_FEATURE(kSecurityKeyAttestationPrompt,
 // TODO(alexmos): Move this and the other site isolation features below to
 // browser_features, as they are only used on the browser side.
 BASE_FEATURE(kSitePerProcess,
-             "SitePerProcess",                  // enabled by default
-             base::FEATURE_ENABLED_BY_DEFAULT   // in Bromite
+             "SitePerProcess",
+#if BUILDFLAG(IS_ANDROID)
+             base::FEATURE_DISABLED_BY_DEFAULT
+#else
+             base::FEATURE_ENABLED_BY_DEFAULT
+#endif
 );
 
 #if BUILDFLAG(IS_CHROMEOS_ASH)
diff --git a/components/site_isolation/features.cc b/components/site_isolation/features.cc
--- a/components/site_isolation/features.cc
+++ b/components/site_isolation/features.cc
@@ -64,8 +64,8 @@ BASE_FEATURE(kSiteIsolationForOAuthSites,
 // kSitePerProcess won't be checked at all, and field trials won't activate
 // either the control or the experiment group.
 BASE_FEATURE(kSiteIsolationMemoryThresholds,
-             "SiteIsolationMemoryThresholds",     // must be enabled
-             base::FEATURE_ENABLED_BY_DEFAULT);   // in Bromite
+             "SiteIsolationMemoryThresholds",
+             base::FEATURE_DISABLED_BY_DEFAULT);
 const char kStrictSiteIsolationMemoryThresholdParamName[] =
     "strict_site_isolation_threshold_mb";
 const char kPartialSiteIsolationMemoryThresholdParamName[] =
-- 
2.34.1

